frontend/src/metabase/visualizations/components/TableInteractive.jsx

```diff
@@ -320,7 +321,7 @@ export default class TableInteractive extends Component<*, Props, State> {
                     className={cx("cellData", { "cursor-pointer": isSortable })}
                     onClick={isSortable && this.setSort.bind(this, column)}
                 >
-                    {columnTitle}
+                    <Ellipsified> {columnTitle} </Ellipsified>
```

Comment:
One other small thing would be removing the extra spaces in between the components here and on line 285.